common/sgx/eeid.c

```diff
@@ -438,15 +448,81 @@ static oe_result_t _verify_signature(
 
     result = OE_OK;
 
+done:
+    return result;
+}
+#else
+#ifdef _WIN32
+static oe_result_t _verify_signature(
+    const OE_SHA256* msg_hsh,
+    const uint8_t* modulus,
+    const uint8_t* exponent,
+    const uint8_t* signature)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    oe_rsa_public_key_t pk;
+
+    BCRYPT_ALG_HANDLE hAlgorithm;
```

Comment:
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md#naming-conventions says:

> Use lower_snake_case for variable, member/field, and function names.

That means no Hungarian.